test/poincare.jl

```diff
@@ -96,4 +96,5 @@ end
     A = poincaresos(X, plane1)
     @test dimension(A) == 3
     @test all(x -> abs(x) < 1e-12, A[:, 1])
+    B = poincaresos(X, plane2)
```

Comment:
But this doesn't test anything, at least test that `@test dimension(B) == 3`.

Reply:
Yeah I wasn't sure what to test so I just put it in there so it would raise an error if it doesn't work. Anyway, I've added some actual tests